src/engine/objects/resolvers/visitors.rs

```diff
@@ -1039,7 +1155,19 @@ where
             .await;
 
         if let Some(handlers) = context.event_handlers().after_rel_delete(&rel_label) {
-            handlers.iter().try_fold(rels, |v, f| f(v))?;
+            for f in handlers.iter() {
+                rels = f(
+                    rels,
+                    EventFacade::new(
+                        // TODO: where do we get rel_name
```

Comment:
The RelQueryVar struct has a function to obtain it, so it'd likely be rel_var.label().

Reply:
Fixed.